pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -422,11 +428,22 @@ func (l *LibvirtDomainManager) generateCloudInitISO(vmi *v1.VirtualMachineInstan
 // The Domain.Spec can be alterned in this function and any changes
 // made to the domain will get set in libvirt after this function exits.
 func (l *LibvirtDomainManager) preStartHook(vmi *v1.VirtualMachineInstance, domain *api.Domain) (*api.Domain, error) {
-
 	logger := log.Log.Object(vmi)
 
 	logger.Info("Executing PreStartHook on VMI pod environment")
 
+	disksInfo := map[string]*containerdisk.DiskInfo{}
```

Comment:
Would it make sense to just stay with the `grpc` DiskInfo types here? I can see value when for instance thinking about different GRPC versions, but maybe we should only introduce this extra conversion once we actuallly have to bump the grpc structs?

Reply:
That would require to propagate the grpc DiskInfo type to `containerdisk` package (more precisely to `containerdisk.CreateEphemeralImages` function). With the conversion I tried to avoid such kind of dependency (besides I think it will cause confusion since there is already a DiskInfo type in the containerdisk package).

Reply:
:+1: